docs/specs/build/config/basics.yml

```diff
@@ -71,4 +71,94 @@ inputs:
   docs/codesnippet/index.md:
 outputs:
   docs/index.json: |
-    { "content": "" }
\ No newline at end of file
+    { "content": "" }
+---
+# Global metadata
+inputs:
+  docfx.yml: |
+    globalMetadata:
+      key: value
+  docs/a.md:
+  docs/b.md:
+outputs:
+  docs/a.json: |
+    { "content": "" }
+  docs/a.mta.json: |
+    { "key": "value" }
+  docs/b.json: |
+    { "content": "" }
+  docs/b.mta.json: |
+    { "key": "value" }
+---
+# File metadata of object form
+inputs:
+  docfx.yml: |
+    fileMetadata:
+      docs/:
+        key: value
+      docs/index.md:
+        key: another value
+  docs/a.md:
+  docs/b.md:
+  docs/index.md:
+outputs:
+  docs/a.json: |
+    { "content": "" }
+  docs/a.mta.json: |
```

Comment:
if we are producing current ops `*.page.json` format, can `a.mta.json` be merged into `a.json`:

{ "content": "", "metadata": { key: value } }